pkg/ztunnel/zds/server_test.go

```diff
@@ -9,14 +9,11 @@ import (
 	"net"
```

Comment:
Lets drop the changes to this, any `zds` related changes can be handled in a `zds` related pr.

Reply:
ACK removing this, I was initially removing all references to endpoint manager. We can do that in a ZDS specific PR